gui/webdisplay/src/TWebWindowsManager.cxx

```diff
@@ -196,6 +202,12 @@ bool ROOT::Experimental::TWebWindowsManager::CreateHttpServer(bool with_http)
       else if (serv_thrd && strstr(serv_thrd, "no"))
          fUseHttpThrd = false;
 
+      const char *send_thrds = gEnv->GetValue("WebGui.SenderThrds", "");
+      if (send_thrds && strstr(send_thrds, "yes"))
+         fUseSenderThreads = true;
+      else if (send_thrds && strstr(send_thrds, "no"))
+         fUseSenderThreads = false;
```

Comment:
else? The behavior seems indeterministic; I'd prefer "SenderThrds" to have a default, and adding an error here if `gEnv->GetValue("WebGui.SenderThrds", "yes");` (or whatever the default will be) returns something other than "yes" or "no". Think `WebGui.SenderThrds: on` :-)

Reply:
Otherwise remains default

Reply:
IIUC `fUseSenderThreads` remains untouched, which is the *previous* value (which might or might not be the default, it can get changed programmatically, right?)

I'd still like to at least see an error for `WebGui.SenderThrds: on`!

Reply:
Done